packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -0,0 +1,299 @@
+# Mapping Table Column
+
+## Overview
+
+The `nimble-table-column-mapping` is a component that supports rendering specific number, boolean, or string values as mapped icons, spinners, or text. `nimble-table-column-icon` is a specialized version of `nimble-table-column-mapping` which defaults to having a minimal, fixed width. The actual mappings are defined by child elements `nimble-mapping-icon`, `nimble-mapping-spinner`, and `nimble-mapping-text`.
```

Comment:
> which defaults to having a minimal, fixed width

My understanding is that we are going to be stricter than just defaulting the icon column to a fixed width -- we wouldn't even support resizing that column, so the "default" is really just the only behavior associated with the column. I think it would be good to be explicit about that here since you mention its sizing behavior. Alternatively, I think if you just want to discuss the sizing behavior below, then it might be better to not talking about it at all here and just saying something like _`nimble-table-column-icon` is a specialized version of `nimble-table-column-mapping` that supports showing a single icon or spinner._

Reply:
I hadn't thought we would prevent resizing. I figured we would support text column headers for icon columns, in which case a client would want to set the size such that the header was fully visible. You mention in a later comment that icon column headers should probably also be icons. That seems overly restrictive to me. I'd worry about the ability to communicate a column's meaning via an icon. I can go that direction for now, but I'd like to get confirmation from Leslie.

Reply:
I'm leaving another comment thread later in the PR open to track the issue of whether icon columns should be resizable.